### DISCUSSION TITLE
Pin column ?

### DISCUSSION THREAD

**Discussion by Nikhilchandra-Rathod** [icflorescu/mantine-datatable#242] [Q&A]
How can I pin column to left side ?

**Comment by icflorescu** [comment]
Duplicate of https://github.com/icflorescu/mantine-datatable/discussions/229.
Please read my comment in that discussion. Like I wrote, this is not exactly a breeze to implement and I'm more than open to a constructive discussion on how this should be implemented (or better yet, a PR).
